pwndbg/gdblib/disasm/aarch64.py

```diff
@@ -108,6 +160,27 @@ def __init__(self, architecture: str) -> None:
             ARM64_INS_CCMN: self._common_cmp_annotator_builder("cpsr", ""),
         }
 
+    @override
+    def _set_annotation_string(self, instruction: PwndbgInstruction, emu: Emulator) -> None:
+        # Dispatch to the correct handler
+        if instruction.id in AARCH64_SINGLE_LOAD_INSTRUCTIONS:
+            # TODO: replace the 8 with actual read size from other PR
```

Comment:
So this will be done in another PR?

Reply:
Good catch - this was a small change I forgot to fill in after a rebase. Fixed it.